mqtt-sn-serial-bridge.c

```diff
@@ -53,11 +53,65 @@ uint8_t frwdencap = FALSE;
 
 uint8_t keep_running = TRUE;
 
+static speed_t baud_lookup(int baud) {
```

Comment:
It is weird that there isn't an OS function for doing this.
Could there be a single lookup table for these, rather than having to map them all twice?